nexus/reconfigurator/planning/src/blueprint_builder/external_networking.rs

```diff
@@ -371,8 +371,7 @@ impl<'a> BuilderExternalNetworking<'a> {
     /// which could otherwise only be added via RSS.
     ///
     /// TODO-cleanup: Remove when external DNS addresses are in the policy.
-    #[cfg(test)]
-    pub fn add_external_dns_ip(&mut self, addr: IpAddr) {
+    pub(crate) fn add_external_dns_ip(&mut self, addr: IpAddr) {
         assert!(
```

Comment:
Same note as above - if this isn't `#[cfg(test)]` it needs to be fallible instead of asserting.

Reply:
Fixed.